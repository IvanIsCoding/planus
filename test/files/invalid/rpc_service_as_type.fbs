table Foo {
  x: Bar;
}

rpc_service Bar {
  x(Foo): Foo;
}