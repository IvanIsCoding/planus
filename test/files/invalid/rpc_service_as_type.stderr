[0m[1m[38;5;9merror[0m[1m: Rpc services are not currently supported[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/rpc_service_as_type.fbs:5:1
  [0m[34mâ”‚[0m
[0m[34m5[0m [0m[34mâ”‚[0m [0m[31mrpc_service Bar[0m {
  [0m[34mâ”‚[0m [0m[31m^^^^^^^^^^^^^^^[0m

[0m[1m[38;5;9merror[0m[1m: Cannot use the RpcService Bar in a type context[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/rpc_service_as_type.fbs:2:6
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [0m[31mBar[0m;
  [0m[34mâ”‚[0m      [0m[31m^^^[0m [0m[31mRpc services cannot be used as types[0m
  [0m[34mÂ·[0m
[0m[34m5[0m [0m[34mâ”‚[0m rpc_service Bar {
  [0m[34mâ”‚[0m [0m[34m---------------[0m [0m[34mService was defined here[0m

Error: Bailing because of errors
